src/containers/greeting/Greeting.js

```diff
@@ -1,56 +1,104 @@
 import React from "react";
 import "./Greeting.css";
 import SocialMedia from "../../components/socialMedia/SocialMedia";
-import Button from '../../components/button/Button';
-import { greeting } from "../../portfolio";
+import Button from "../../components/button/Button";
+import { greeting, settings } from "../../portfolio";
 import { Fade } from "react-reveal";
-import FeelingProud from './FeelingProud';
+import FeelingProud from "./FeelingProud";
 
 export default function Greeting(props) {
   const theme = props.theme;
-  return (
-		<Fade bottom duration={2000} distance="40px">
-			<div className="greet-main" id="greeting">
-				<div className="greeting-main">
-					<div className="greeting-text-div">
-						<div>
-							<h1 className="greeting-text" style={{ color: theme.text }}>
-								{greeting.title}
-							</h1>
-							<h2 className="greeting-nickname" style={{ color: theme.text }}>
-								( {greeting.nickname} )
-							</h2>
-							<p
-								className="greeting-text-p subTitle"
-								style={{ color: theme.secondaryText }}
-							>
-								{greeting.subTitle}
-							</p>
-							<SocialMedia />
-							<div className="portfolio-repo-btn-div">
-								<Button
-									text="⭐ Star Me On Github"
-									newTab={true}
-									href={greeting.portfolio_repository}
-									theme={theme}
-									className="portfolio-repo-btn"
-								/>
-							</div>
-							{/* <div className="button-greeting-div">
+  if (settings.isNickName) {
```

Comment:
I think it's not a good idea to repeat the whole code. We can do like below.
Build seperate functional component which take isNickName as prop and give result as <h2> containing nickname. If isNickName is false then return empty <></>

Reply:
Got it , i will try that.